tools/GenTypeInfo.hs

```diff
@@ -195,7 +195,9 @@ daterange
 _daterange
 int8range
 _int8range
-|]
+citext
+_citext                  array_citext
+|] -- Note: You must enable the citext extension (CREATE EXTENSION IF NOT EXISTS citext) to generate the citext values
```

Comment:
I could make the script do this automatically as well

Reply:
You can't do this, because the type oid of citext is not stable from database to database.  You need to remove this from the static typeinfo, and should be using the textual name for type checking instead.  See the documentation in the FromField module, and the hstore instances for an example.

Reply:
Ok, I’ll remove this